definitions/browser-application/definition.yml

```diff
@@ -7,3 +7,21 @@ compositeMetrics:
 configuration:
   entityExpirationTime: QUARTERLY
   alertable: true
+
+# Select entityGuid, requestUrl from AjaxRequest where requestUrl is not null
+relationships:
```

Comment:
This relationship section could grow a bit right? (CALLS, IS, etc) would it make sense to keep it as a separate file on the folder? with maybe just some configuration on the main definition file.

Reply:
ya thats an option!

Reply:
done! Although I'll skip any config in definition file

Reply:
thanks!